cmd/kuberhealthy/reflector.go

```diff
@@ -51,8 +51,8 @@ func (sr *StateReflector) Start() {
 	sr.reflector.Run(sr.reflectorSigChan)
 }
 
-// CurrentStatuses returns the current summary of all checks as known by the cache
-func (sr *StateReflector) CurrentStatus() health.State {
+// CurrentStatus returns the current summary of all checks as known by the cache
```

Comment:
What happens if I want to pull the state for all checks?  Do I use a blank input parameter?  How do I know to do that?

Add some details to the comment here about how you can use this to poll everything.